ui/page/dcrdex/market.go

```diff
@@ -506,6 +528,10 @@ func (pg *DEXMarketPage) OnNavigatedFrom() {
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *DEXMarketPage) Layout(gtx C) D {
+	if pg.isDEXReset(true) {
+		return D{}
+	}
```

Comment:
If this is the only place you pass true, then you can bring the CloseCurrentPage code here and you don't need to pass any param to the method.